system-test/storage.ts

```diff
@@ -1433,19 +1274,14 @@ describe('storage', () => {
         });
       }
 
-      function deleteFiles(callback: async.ErrorCallback) {
-        async.each(
-          FILES,
-          (file, next) => {
-            file.setMetadata({temporaryHold: null}, (err: ApiError | null) => {
-              if (err) {
-                next(err);
-                return;
-              }
-              file.delete(next as DeleteFileCallback);
-            });
-          },
-          callback
+      async function deleteFilesAsync() {
+        await new Promise(resolve =>
+          setTimeout(resolve, RETENTION_PERIOD_SECONDS * 1000)
+        );
+        return Promise.all(
+          FILES.map(file =>
+            file.setMetadata({temporaryHold: null}).then(() => file.delete())
```

Comment:
The mix of promises and async/await is weird.  Can we just use an async function here and awaits?

Reply:
Changed that, thanks